src/Shared/Microsoft.AspNetCore.Razor.LanguageSupport/Index.cs

```diff
@@ -146,7 +146,7 @@ public override string ToString()
 
     [DoesNotReturn]
     private static void ThrowValueArgumentOutOfRange_NeedNonNegNumException()
-        => throw new ArgumentOutOfRangeException("value", SR.ArgumentOutOfRange_NeedNonNegNum);
```

Comment:
I assume this wasn't exposed publicly, which is why we don't need it localized?

Reply:
I mean, exception text always has the potential to be exposed to the user. I was assuming that the odds of this particular exception being thrown from production code made it less important and not worth exploding into many resources dlls.

Reply:
AFAIK when this repo was owned by ASP.NET there was zero localization, so there are probably lots of gaps still, and nobody should assume that the current situation is designed in any way :P